models/issues/issue_lock.go

```diff
@@ -64,3 +67,13 @@ func updateIssueLock(ctx context.Context, opts *IssueLockOptions, lock bool) err
 
 	return committer.Commit()
 }
+
+// IsValidReason checks to make sure that the reason submitted
+// matches any of the values in the config
+func IsValidReason(reason string) bool {
```

Comment:
Not sure if it adds too much complexity, but it might be worth looking into integrating `strings.EqualFold` as then you don't need to worry about converting cases.